WebRoot/public/login.jsp

```diff
@@ -224,16 +224,16 @@
                                 String versionPart = tokenParts[2];
                                 Map<?, ?> decodedTokenMap = TokenUtil.getAttrs(versionPart);
                                 String version = (String) decodedTokenMap.get("version");
-                                pageContext.setAttribute("isZ9Mailbox", version.startsWith("9"));
+                                pageContext.setAttribute("isUpgradedMailbox", version.startsWith("9") || version.startsWith("10"));
```

Comment:
version greater than >=9 by getting major so no need to update in future for version 11?

Reply:
it's java code so I was not sure how to do that, let me check with someone

Reply:
boolean condition = Integer.parseInt(version) >= 9 ? true : false;
pageContext.setAttribute("isUpgradedMailbox", condition);

Reply:
Fixed